src/components/ChecToast.vue

```diff
@@ -34,17 +45,27 @@ export default {
       },
     },
     /**
-     * Whether the notification should not automatically disappear after a certain period of time (`hideTime`)
-     */
+         * Whether the notification should not automatically disappear after a certain period of time (`hideTime`)
+         */
     persist: Boolean,
     /**
-     * The time in milliseconds after which a notification will disappear, if `persist` is true
-     */
+         * The time in milliseconds after which a notification will disappear, if `persist` is true
+         */
     hideTime: {
       type: Number,
       default: 5000, // milliseconds
     },
   },
+  data() {
+    return {
+      icon: {
```

Comment:
This is a constant and shouldn't be defined in `data`. Variables created here will be reactive, which is unnecessary for this.

Reply:
How I'd recommend you change this is just create a `computed` that does something like:

```
const icons = {...};
return icons[this.variant];
```